CONTRIBUTING.rst

```diff
@@ -86,17 +86,50 @@ Documentation
 
      First line of new section.
 - If you add a new feature, demonstrate its awesomeness in the `examples page`_!
-- If your change is noteworthy, add an entry to the changelog_.
-  Use `semantic newlines`_, and add a link to your pull request:
+
+
+Changelog
+^^^^^^^^^
+
+If your change is noteworthy, there needs to be a changelog entry, so our users can learn about it!
+
+To avoid merge conflicts, we use the towncrier_ package to manage our changelog.
+``towncrier`` uses independent files for each pull request -- so called *news fragments* -- instead one monolithic changelog file.
+On release, those news fragments are compiled into our ``CHANGELOG.rst``.
+
+You don't need to install ``towncrier`` yourself, you just have to abide to a few simple rules:
+
+- For each pull request, add a new file into ``changelog.d`` with a filename adhering to the ``pr#.change|deprecation|breaking.rst`` schema:
+  For example ``changelog.d/42.change.rst`` for a non-breaking change, that is proposed in pull request number 42.
+- As with other docs, please use `semantic newlines`_ within news fragments.
+- Wrap symbols like modules, functions, or classes into double backticks so they are rendered in a monospaced font.
+- If you mention functions or other callables, add parantheses at the end of their names: ``attr.func()`` or ``attr.Class.method()``.
+  This makes the changelog a lot more readable.
+- Prefer simple past or constructions with "now".
+  For example:
+
+  + Added ``attr.validators.func()``.
```

Comment:
I’ve moved away from present tense in changelogs.  It reads weird and is kind of just the remnant of using present tense in commit messages.  And changelogs are for users, so I apply different rules there.